Misc/_posts/2020-07-14-Laser-1-Factoring.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+title: "Laser-1-Factoring"
```

Comment:
You can remove the `-` from the title if you want

Reply:
Yass